doc/get-started.md

```diff
@@ -75,8 +75,6 @@ dependencies:
 To use the Flutter plugin on Android devices, you need to configure Gradle so it
 can download the Reader SDK package properly and make it available to the
 Flutter library as a resource. The key installation steps are outlined below.
-For more information on installing Reader SDK for Android, see the
```

Comment:
We don't need this and it can mislead developer to do extra configuration. 

feedback from Daniel: 
> Feedback suggestion: I got mislead into thinking that I needed to follow the full Android setup instructions after following the link to that page

Reply:
I thought they *did* have to follow those instructions? Is it that they only have follow the first few steps to set things up and not realizing they can stop? We could fix that by pulling those steps out into a stand-alone installation guide...

Reply:
All the necessary steps are in this doc already and developer don't need go to docs site. So they don't need follow instructions on our docs site if they follow this, and they don't know when to stop and add extra code in the build.gradle and failed.

I think it's good to keep a reference to them, but putting it here is likely to mislead developers.

Reply:
Makes sense. Just wanted to confirm we weren't losing necessary info. LGTM